pyEpiabm/pyEpiabm/tests/test_sweep/test_spatial_sweep.py

```diff
@@ -45,7 +48,7 @@ def test__call__(self, mock_inf, mock_force, mock_poisson, mock_dist,
         mock_inf.return_value = 10.0
         mock_force.return_value = 100.0
         time = 1
-        Parameters.instance().infection_radius = 5000
+        Parameters.instance().infection_radius = 1000
```

Comment:
Do we have a check that no infections occur when this is really small?

Reply:
Yep, it's like the penultimate bit